plutus-use-cases/test/Spec/Auction.hs

```diff
@@ -42,12 +42,12 @@ import           Test.QuickCheck                    hiding ((.&&.))
 import           Test.Tasty
 import           Test.Tasty.QuickCheck              (testProperty)
 
-params :: AuctionParams
-params =
+params :: POSIXTime -> AuctionParams
```

Comment:
Same here - we can just choose a start time

Reply:
The rational behind passing around the startTime was to ensure that the emulator and the contract use the same `startTime`. For example, suppose that for the function `params` we use a constant start time of `1501010` instead of passing in a parameter. The problem with that is that if we use the `checkPredicate` function in testing, it uses the default `SlotConfig`, so there will be a conflict in the `startTime`. Of course, an alternative will be to use `checkPredicateOptions` instead by using our custom startTime. Would you rather we do that instead of passing the startTime around as a parameter?

Reply:
@j-mueller Which of the two options would you prefer?

Reply:
hm, good point. Ok let's leave it as it is.